src/storage/src/table/state_table.rs

```diff
@@ -45,13 +45,12 @@ pub struct StateTable<S: StateStore> {
     order_types: Vec<OrderType>,
 
     /// buffer key/values
-    mem_table: MemTable,
+    pub mem_table: MemTable,
 
     /// Relation layer
     cell_based_table: CellBasedTable<S>,
 
-    #[expect(dead_code)]
-    pk_indices: Vec<usize>,
+    pub pk_indices: Vec<usize>,
```

Comment:
same

Reply:
Thanks for your comments and I will refactor this.